### DISCUSSION TITLE
Malicious Links are scripted inside the head section.

### DISCUSSION THREAD

**Discussion by wsrmkting** [elementor/elementor#24829] [Developers Community]
### Start a discussion

Hello!

Hope you're having a fantastic day.

I manage Google ads on a daily basis, and typically, everything runs smoothly. However, I recently received notification from Google about the presence of a few malicious links on my landing page. In response, I contacted Elementor Support, my go-to for web design using Elementor. After delving into WordPress and conducting thorough searches, they identified the hidden malicious links embedded in the header section. Their recommendation was to eliminate them using custom code.

Locating these links is challenging as most malicious links are well hidden deep as they advised. Unfortunately, in adherence to their terms and conditions, Elementor Support couldn't extend further assistance beyond this point. They directed me to seek help from the Elementor developers on GitHub. While I manage the website, I lack proficiency in coding. Is there anyone who could provide assistance?

For your reference, please refer to the two attached images accompanying this message below.

Best regards,
James

![2037c8611a4bf29151f361dd4a91fe70](https://github.com/elementor/elementor/assets/156973212/a9c64c7a-ebe1-4216-930a-41a0cde77665)
<img width="866" alt="Screenshot 2024-01-18 at 9 44 42 am" src="https://github.com/elementor/elementor/assets/156973212/55a3d2bb-dba2-437c-9d5c-a7f2f00a7a18">

**Comment by perks98** [comment]
@wsrmkting Are you saying your website is compromised? If so get a security plugin and run a scan on the site to see if that can detect any abnormal code. Reset the password for all editor and above access. Make sure to reinstall WordPress and reinstall all plugins and themes. If your hosting provider has a malware scanner also run that. Also make sure to reset the database password and change it in the config file (maybe your host can help with that).

Code can come from all sorts of sources, so script can inject more script. Maybe try to download your whole website and use a code editor like Visual Code and search the files for the code Google has referenced to see if you can locate it and then remove it on the hosting.

I don't think people from Elementor developers community on github can help, I think you need someone to assist you by logging in to your website and hosting to investigate which isn't viable to strangers on a community. That's just my opinion, but others may be able to help more if they reply in this thread. If you don't have any luck from re-installing plugins, themes and WordPress and changing passwords, consider commissioning someone to assist or creating a fresh hosting and import as a template so you know it should be clean then. Maybe ask on Elementor Facebook maybe?

I got this from my free desktop anti-virus when visiting the site:
![Screenshot_975](https://github.com/elementor/elementor/assets/55829113/08272a6e-2992-482a-9d36-5a937cfabd4c)

**Comment by wsrmkting** [comment]
Appreciate your prompt response, @perks98. From my end, the website seems to be functioning normally. The link issue may be specific to certain desktop setups or alternate browsers. I've turned to this platform based on the recommendation of Elementor Support. I plan to explore the Elementor Facebook Community next.
